### ISSUE TITLE
Installation is different on Mac & Arduino 1.0.5

### ISSUE THREAD

**Issue by nhimf** [sudar/arduino-extra-cores#5]
I have tried to install extra cores to my Arduino setup, but I had to take a few different steps:

I copied the files to Arduino.app/Contents/Resources/Java/hardware/arduino except for the boards.txt file. This one I had to concatenate to the boards.txt already in place.
"cat boards.txt >> Arduino.app/Contents/Resources/Java/hardware/arduino/boards.txt"

Also I had to removed the line "atmega16-8.upload.protocol=stk500v2" from boards.txt, because this overruled my setting of the programmer in Arduino itself.

For now I could at least program my atmega16a with an usbasp :-)

Perhaps you could add this to your Installation section.

**Comment by nhimf** [comment]
I have done a little search on this issue and it seems you have to place everything under a subdir within the hardware dir.
The above works, but hard to keep when you update. Maybe a little clarification that you have to copy the subdir (arduino-extra-cores) to the hardware dir or create a subdir yourself would help.

**Comment by sudar** [comment]
@nhimf Instead of copying the folder into the hardware folder inside your Arduino installation, you can also create a hardware/ folder inside your sketchbook folder and copy arduino-extra-core folder inside it.

**Comment by nhimf** [comment]
@sudar Yup, that is what I eventually have done. First I had copied the files directly into the hardware folder, but that didn't work :-)

**Comment by sudar** [comment]
All the files of this repo should be inside `hardware/arduino-extra-cores` folder inside your sketchbook folder. 

What is the error you are getting?

**Comment by nhimf** [comment]
Now I am getting none, after I realized I had to put everything in a subfolder. For me it wasn't clear at first that I had to put it in hardware/<some subdir>. After a little searching around on the net I found what I was doing wrong.

So maybe it is useful  for others if you add an extra clarification to the readme that everything has to be in a subfolder (e.g. the arduino-extra-cores).
Op 13 jul. 2013, om 23:15 heeft Sudar notifications@github.com het volgende geschreven:
